lib/param_parsing/versiontf.go

```diff
@@ -12,7 +12,7 @@ import (
 
 func GetVersionFromVersionsTF(params Params) (Params, error) {
 	var tfConstraints []string
-	var exactConstraints []string
+	//var exactConstraints []string
```

Comment:
We probably can drop this as it is not used anymore

Reply:
Thanks for finding. Deleted.